pyproject.toml

```diff
@@ -15,9 +15,10 @@ singer-sdk = "^0.3.4"
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.1.2"
-mypy = "^0.812"
+mypy = "^0.910"
 flake8 = "^3.9.2"
 black = {version = "^21.5b1", allow-prereleases = true}
+types-requests = "^2.25.6"
```

Comment:
@laurentS - Should we install this type stub package also in the sdk core package?

cc @edgarrmondragon

Reply:
if the template relies on `requests`, then yes, as `mypy` will be looking for the type declarations as well.